lib/Cpdf.php

```diff
@@ -2975,6 +3062,137 @@ protected function o_embedded_file($id, $action, $options = null): ?string
         return null;
     }
 
+    /**
+     * Enable PDF/A compliance mode
+     */
+    public function enablePdfACompliance()
+    {
+        $this->pdfa = true;
+
+        $iccProfilePath = __DIR__ . '/res/sRGB2014.icc';
+
+        $this->o_catalog($this->catalogId, 'outputIntents', [
+            'iccProfileData' => file_get_contents($iccProfilePath),
+            'iccProfileName' => 'sRGB2014.icc',
```

Comment:
should this be basename
```suggestion
            'iccProfileName' => basename($iccProfilePath),
```